src/pages/wallet-overview/wallet-overview.spec.ts

```diff
@@ -23,17 +38,239 @@ describe("WalletOverviewPage", () => {
+  })
+
+  it("should have an empty amounts for the dougnut chart", () => {
+    expect(comp.currenciesForDoughnutCanvas.length).toEqual(0);
+  })
+
+  it("should have empty currencies for the dougnut chart", () => {
+    expect(comp.currenciesForDoughnutCanvasCurrencies.length).toEqual(0);
+  })
+
+  it("should have current wallet as undefined ", () => {
+    expect(comp.currentWallet).toBeUndefined();
+  })
+
+  it("should have current wallet index as 0", () => {
+    expect(comp.currentWalletIndex).toEqual(0);
+  })
+
+  it("should have an empty legend list", () => {
+    expect(comp.legendList.length).toEqual(0);
+  })
+
+  it("should have no available currencies", () => {
+    expect(comp.availableCurrencies.length).toEqual(0);
+  })
+
+  it("should have show fund status to be truthy", () => {
+    expect(comp.showFundsStatus).toBeTruthy();
+  })
+
+  it("should have two factor authentication to be falsy", () => {
+    expect(comp.twoFactorStatus).toBeFalsy();
+  })
+
+  it("should have funds visibility standard to 'shown'", () => {
+    expect(comp.walletFundsVisibility).toEqual("shown");
+  })
+
+  it("should have visibility hidden after clicking the show funds switch", () => {
+    comp.fundsSwitch();
+
+    expect(comp.walletFundsVisibility).toEqual("hidden");
+  })
+
+  it("should have show funds boolean to be falsty after clicking the show funds switch", () => {
+    comp.fundsSwitch();
+
+    expect(comp.showFundsStatus).toBeFalsy();
+  })
+
+  it("should show two factor authentication to be falsy after clicking the two factor switch", () => {
+    comp.twoFactorStatusSwitch();
+
+    expect(comp.twoFactorStatus).toBeTruthy();
+  })
+
+  it("should return null after clicking the delete wallet click without wallets", () => {
+    let result = comp.deleteWalletClick();
+
+    expect(result).toBeNull();
+  })
+
+  it("should return false after deleting a wallet that doesn't exist", () => {
+    let result = comp.deleteSelectedWallet("I DON'T EXIST");
+
+    expect(result).toBeFalsy();
+  })
+
+  it("should return false after getting all wallets because there are no wallets", () => {
+    comp.getAllWallets();
+
+    expect(comp.currentWallet).toBeUndefined();
+  })
+
+  it("should have no available currencies after getting them because there are none defined", () => {
+    comp.getAvailableCurrencies();
+
+    expect(comp.availableCurrencies.length).toEqual(0);
+  })
+
+  it("should return an empty array after getting the wallet currencies because the wallet does not exist", (done) => {
+    comp.getWalletCurrency("I DON'T EXIST").then(data => {
+      expect(data).toEqual([]);
+      done();
+    });
+  })
+
+  it("should return false after calculating the picked currency values because there is no picked currency", (done) => {
+    comp.setCalculatedCurrencyValue().then(data => {
+      expect(data).toEqual([]);
+      done();
+    });
+  })
+
+  it("should open wallet overview page correctly", () => {
```

Comment:
What is being tested here? The title suggests we are testing if opening the wallet overview page works. However the tests open the landing page.